packages/core/core/src/config.ts

```diff
@@ -91,6 +91,8 @@ export class Config {
 
   client: string;
 
+  webpackConfigName: string;
```

Comment:
It doesn't "feel right"  to hardwire this into core `config`, `plugin`, `plugins`, and `engine` while making the plugin optional.

I realize there may not be another way forward that doesn't involve a heavy refactor, but hopefully we can rethink this in the near future.

Reply:
I agree is not ideal, it's being done this way to remove the options object that was being passed with this info. there is no way to otherwise without a refactor outside the scope of this PR (namely the config+cmd refactor)